### PR TITLE
Update readme FAQs

### PR DESCRIPTION
**Pull request by pamelafox**
## Purpose

This PR updates the readme with some FAQs and rewording based on recent support issues.

If we're worried about the FAQs adding too much length to the readme, we could use HTML details/summary elements (GitHub supports!) so that only the questions are visible by default.

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[X] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -125,8 +131,11 @@ Once in the web app:
 
 ***Answer***: Chunking allows us to limit the amount of information we send to OpenAI due to token limits. By breaking up the content, it allows us to easily find potential chunks of text that we can inject into OpenAI. The method of chunking we use leverages a sliding window of text such that sentences that end one chunk will start the next. This allows us to reduce the chance of losing the context of the text.
 
-### Troubleshooting
+***Question***: How can we upload additional PDFs without redeploying everything?
+
+***Answer***: To upload more PDFs, put them in the data/ folder and run `./scripts/prepdocs.sh` or `./scripts/prepdocs.ps1`. To avoid reuploading existing docs, move them out of the data folder. You could also implement checks to see whats been uploaded before; our code doesn't yet have such checks.
 
-If you see this error while running `azd deploy`: `read /tmp/azd1992237260/backend_env/lib64: is a directory`, then delete the `./app/backend/backend_env folder` and re-run the `azd deploy` command.  This issue is being tracked here: <https://github.com/Azure/azure-dev/issues/1237>
```
Comment (pamelafox) [inline README.md:130]: @vhvb1989 Is it okay to remove this now? The issue seems to have been closed a few months ago.

File: README.md [hunk 2]
```diff
@@ -125,8 +131,11 @@ Once in the web app:
 
 ***Answer***: Chunking allows us to limit the amount of information we send to OpenAI due to token limits. By breaking up the content, it allows us to easily find potential chunks of text that we can inject into OpenAI. The method of chunking we use leverages a sliding window of text such that sentences that end one chunk will start the next. This allows us to reduce the chance of losing the context of the text.
 
-### Troubleshooting
+***Question***: How can we upload additional PDFs without redeploying everything?
```
Comment (pamelafox) [inline README.md:134]: This is where we could use summary/details elements instead, to hide the answers by default.

**Review by chuwik — APPROVED** [review]

**Comment by chuwik** [comment]
I support using summary/details elements, seems like it'd make the FAQ easier to read.

**Comment by pamelafox** [comment]
@chuwik Sure, changed to summary/details

**Pull request merged**